### PR TITLE
suggest using knative-operators's installation script in minishift doc

### PR DESCRIPTION
**Pull request by vorburger**

### CODE REVIEW

**Comment by vorburger** [comment]
based on #976 with a slight change of course.. better to just link to instead of duplicating scripts, agreed?

@gyliu513 @jcrossley3 @bbrowning

**Comment by vorburger** [comment]
@kameshsampath

File: docs/install/Knative-with-Minishift.md [hunk 1]
```diff
@@ -20,7 +20,25 @@ You can find [guides for other platforms here](README.md).
...
 
-## Configure and start minishift
+## Automatic Set Up
+
+Once you have `minishift` present on your machine and in your `PATH`,
+you can either follow the manual set up steps below,  or you can run the convenient
+scripts from [the openshift-cloud-functions/knative-operators project](https://github.com/openshift-cloud-functions/knative-operators), 
+which do something similiar, like this:
+
+    git clone https://github.com/openshift-cloud-functions/knative-operators
```
Comment (gyliu513) [inline docs/install/Knative-with-Minishift.md]: I prefer following to keep consistent.

```shell
git clone https://github.com/openshift-cloud-functions/knative-operators
cd knative-operators
./etc/scripts/install-on-minishift.sh 
```

File: docs/install/Knative-with-Minishift.md [hunk 2]
```diff
@@ -20,7 +20,25 @@ You can find [guides for other platforms here](README.md).
...
+    cd knative-operators
+    ./etc/scripts/install-on-minishift.sh 
+
+The `myproject` this created is now ready for Knative!  (If you use `oc new-project yourproject`
+to create additional projects, make sure that you apply the x2 `oc adm policy ...` commands from below.)
```
Comment (gyliu513) [inline docs/install/Knative-with-Minishift.md]: what does `x2` mean here? Can you make it more easy to understand?

File: docs/install/Knative-with-Minishift.md (hunk 1 above)
Comment (vorburger) [reply docs/install/Knative-with-Minishift.md]: @gyliu513 sorry but isn't that exactly how it is? I feel like I must be missing something you are trying to point me to... (If you are a commiter on this project, you can click that Insert a suggestion <Ctrl-G> icon, just before the AA / B / i icons, to directly propose a specific modification.

File: docs/install/Knative-with-Minishift.md (hunk 2 above)
Comment (vorburger) [reply docs/install/Knative-with-Minishift.md]: done ("two" instead of "x2")

**Pull request merged**